### ISSUE TITLE
Improve representation of failures for packages not attempted

### ISSUE THREAD

**Issue by gdt** [bsiegert/BulkTracker#34]
Packages won't be built at all if a dep fails.  I'd like to see more information about why as that is the point for bulk-foo.

An example with not enough info is

https://bulktracker.appspot.com/pkg/ag1zfmJ1bGt0cmFja2VyciILEgVidWlsZBiAgICzkv6bCwwLEgNwa2cYgICA0_nBgQsM

**Comment by bsiegert** [comment]
FWIW, I looked up this entry in the DB, and it has no list of failed dependencies. I think (but I may be wrong) that the machine-readable report does not contain that information for `indirect-prefailed` packages.

The original report for that build is at  http://www.ki.nu/pkgsrc/reports/current/NetBSD-9.0/20230812.0412/meta/report.bz2. Looking at it, I see that the report does contain all dependencies in their versions:

```
DEPENDS=python310-3.10.12 mktools-20220614 cwrappers-20220403 cdparanoia-3.0.10.2nb4 flac-1.4.3 flac123-0.0.12nb1 lame-3.100nb5 mpg123-1.31.3 sox-14.4.2nb7 vorbis-tools-1.4.2nb6 jabberd-2.7.0nb17 sirc-2.211nb14 hexchat-2.16.1nb8 fribidi-1.0.13 mariadb-client-10.5.21 mariadb-client-10.6.14 openldap-client-2.6.6 git-2.41.0 git-base-2.41.0nb1 git-docs-2.41.0 mercurial-6.5 emacs25-25.3nb37 emacs-packages-0.12nb1 nvi-1.81.6nb13 vim-gtk3-9.0.1598 xemacs-packages-1.19 qemu-8.0.3 simh-4.0.0.20220501nb5 tme-0.8nb53 wine-5.0.5nb19 ImageMagick-7.1.1.12 gnuplot-5.4.6nb8 netpbm-10.86.34nb4 py310-matplotlib-3.7.2nb1 gawk-5.2.2 php-7.4.33nb2 php-8.0.30 php-8.1.22 php-8.2.9 ruby30-base-3.0.6nb1 ruby31-base-3.1.4nb1 ruby32-base-3.2.2nb2 openjdk8-1.8.382 openjdk11-1.11.0.20.8 tcl-8.6.13 fetchmail-6.4.35 imap-uw-2007fnb11 mailman-2.1.35nb1 mhonarc-2.6.19nb9 mutt-2.2.10nb1 alpine-2.26nb2 spamassassin-4.0.0nb1 bind-9.16.42 clive-2.3.3nb12 nmap-7.94 samba-4.18.5 wpa_supplicant-2.6nb2 xymon-4.3.30nb8 xymonclient-4.3.30nb1 youtube-dl-20211217nb4 lintpkgsrc-2022.09.29 pkglint-23.2.0nb1 ghostscript-agpl-10.01.2 poppler-utils-23.06.0 xpdf-4.04nb8 gnupg-1.4.23nb16 gnupg2-2.4.3 desktop-file-utils-0.26nb2 xe-guest-utilities-7.0.0nb1 xenkernel413-4.13.4nb1 xentools413-4.13.4nb6 ispell-3.4.05 seamonkey-2.53.16nb2 xlockmore-5.72 pigz-2.6 cmake-3.27.1nb1 imake-1.0.9nb1 nasm-2.16.01 yasm-1.3.0 icon-naming-utils-0.8.90nb10 SDL2_mixer-2.0.4nb12 opusfile-0.12nb1 libvorbis-1.3.7 gdbm-1.23 p5-BDB-1.92nb6 p5-BerkeleyDB-0.65nb1 p5-DBD-mysql-4.050nb6 p5-DBD-postgresql-3.14.2nb3 p5-DBI-1.643nb5 py310-bsddb3-6.2.9nb1 py310-gdbm-3.10.12nb1 py310-postgresql-5.1nb1 py310-psycopg2-2.9.7 py310-sqlite3-3.10.12nb24 SDL2-2.28.2 atk-2.38.0 boehm-gc-8.2.2 glib2-2.76.4 libelf-0.8.13nb1 libf2c-20090201nb3 libidn2-2.3.4 libltdl-2.4.7 libusb1-1.0.26 ncurses-6.4 ncursesw-6.4 pango-1.50.14 pcre-8.45 popt-1.19 fuse-2.9.3 perfuse-20190222 SDL2_image-2.6.3 cairo-1.16.0nb9 gdk-pixbuf2-2.42.10nb2 glew-2.2.0nb2 openjpeg-2.5.0nb1 lcms-1.19nb4 lcms2-2.15 librsvg-2.56.0nb1 blas-3.11.0 lapack-3.11.0 boost-1.82.0 ffmpeg4-4.4.4nb1 libogg-1.3.5nb1 libpcap-1.10.4 libsoup-2.74.3nb2 poppler-23.06.0 poppler-includes-23.06.0 cyrus-sasl-2.1.28nb1 openssl-1.1.1u tcp_wrappers-7.6.4 libnotify-0.8.1 gtk3+-3.24.38 motif-2.3.8nb2 tk-8.6.13 suse-13.1nb6 eawpatches-12nb3 mpg123-pulse-1.31.3nb1 pulseaudio-16.1nb4 shared-mime-info-2.2nb2 hicolor-icon-theme-0.17nb1 lua54-stdlib-41.2.2 bulk-small-20230601
```

I think we should be able to show links to these and their status. Thanks for the suggestion!

**Comment by bsiegert** [comment]
I believe that this bug is fixed in the latest BulkTracker version, v2024.4.

You can click on an indirect-failed package and get a list of failed dependencies. Note that builds with a date after today (i.e. starting 2024-04-22) will have more failed dependencies shown: if a package depends on an indirect-failed package, that dependency was incorrectly filtered out until now.
